package-lock.json

```diff
@@ -27,22 +27,13 @@
         "@babel/traverse": "^7.8.4",
         "@babel/types": "^7.8.3",
         "convert-source-map": "^1.7.0",
-        "debug": "^4.1.0",
+        "json5": "^2.1.0",
```

Comment:
Why did this go from `debug` to `json5`?

Reply:
thanks for catching this i've updated this